x-pack/plugins/ml/public/application/explorer/explorer_swimlane.tsx

```diff
@@ -278,59 +304,28 @@ export class ExplorerSwimlane extends React.Component {
       points,
     } = swimlaneData;
 
-    function colorScore(value) {
-      return getSeverityColor(value);
-    }
-
-    const numBuckets = parseInt((endTime - startTime) / stepSecs);
-    const cellHeight = 30;
-    const height = (lanes.length + 1) * cellHeight - 10;
-    const laneLabelWidth = 170;
-
-    element.style('height', `${height + 20}px`);
-    const swimlanes = element.select('.ml-swimlanes');
-    swimlanes.html('');
-
-    const cellWidth = Math.floor((chartWidth / numBuckets) * 100) / 100;
-
-    const xAxisWidth = cellWidth * numBuckets;
-    const xAxisScale = d3.time
-      .scale()
-      .domain([new Date(startTime * 1000), new Date(endTime * 1000)])
-      .range([0, xAxisWidth]);
-
-    // Get the scaled date format to use for x axis tick labels.
-    const timeBuckets = new TimeBuckets();
-    timeBuckets.setInterval(`${stepSecs}s`);
-    const xAxisTickFormat = timeBuckets.getScaledDateFormat();
-
-    function cellMouseOverFactory(time, i) {
-      // Don't use an arrow function here because we need access to `this`,
-      // which is where d3 supplies a reference to the corresponding DOM element.
-      return function(lane) {
-        const bucketScore = getBucketScore(lane, time);
-        if (bucketScore !== 0) {
-          lane = lane === '' ? EMPTY_FIELD_VALUE_LABEL : lane;
-          cellMouseover(this, lane, bucketScore, i, time);
-        }
-      };
-    }
-
-    function cellMouseover(target, laneLabel, bucketScore, index, time) {
+    const cellMouseover = (
+      target: HTMLElement,
+      laneLabel: string,
+      bucketScore: number,
```

Comment:
Might not be related to this PR, but we shouldn't be showing the anomaly score in the tooltip to such high precision:

![image](https://user-images.githubusercontent.com/7405507/80513756-6d888b00-8977-11ea-894a-9e86fd2ee33d.png)

Reply:
Actually I removed `parseInt` by mistake during TS refactoring, just put it back

Reply:
Fixed in https://github.com/elastic/kibana/pull/64056/commits/b1bdde553a4c933eb09adb1d6a6cd860686bb946